volume/local/local_unix.go

```diff
@@ -111,6 +126,13 @@ func (v *localVolume) mount() error {
 	return errors.Wrap(err, "failed to mount local volume")
 }
 
+func (v *localVolume) postMount() error {
+	if v.opts == nil {
+		return nil
+	}
+	return nil
```

Comment:
This code doesn't make much sense

Reply:
Ah, now I see, you actually add some in the following commit

Reply:
What part of it doesn't make sense?
It checks if options are set, and if not, aborts early and does nothing.
Then it checks if a quota size larger than 0 is specified, and if so, either applies it via the quotaCtl, and if none is present, it throws an error about it, because it indicated no quota support is present.
Otherwise it applies the quota.
The reason it's in this postMount function is because actuao Mount is never called for local volumes, since no mounting is needed.

Reply:
Oh god, Github displayed this in the worst way for me (under the code with the following actual code in it).
Yeah, I see what you mean now. The initial nil-check should probably move to the follow-up commit.